Changelog for 0.11.0 release (#183)
QCoDeS/broadbean: docs/changes/0.11.0.rst

Changelog for broadbean 0.11.0
==============================

> The July 2022 release of broadbean with many great improvments for modernizing broadbeans infrastructure.

Reviewer: ```suggestion
The July 2022 release of broadbean with many great improvements for modernizing broadbean's infrastructure.
```

Breaking Changes:
_________________

- Methods and functions  marked for deletion in version `0.10.0` have now been removed. Specifically
  `BluePrint.plot`, `broadbean.bluePrintPlotter`, `Element.plotElement`, `Sequence.plotSequence`
  and `Sequence.plotAWGOutput`. (#107)

New:
____

> - New `Sequance` method outputForSEQXFileWithFlags for setting flags for every element forming a sequence.

Reviewer: ```suggestion
- New `Sequence` method `outputForSEQXFileWithFlags` for setting flags for every element forming a sequence.
```

  This is useful for auxiliary outputs on a Tektronix AWG70000. (#101)

Improved:
_________

> - Fix for invalid identity comparisons in blueprint subomdule. (#102)

Reviewer: ```suggestion
- Fix for invalid identity comparisons in blueprint submodule. (#102)
```
